### PR TITLE
Add context to async example function in Network docs

### PR DESCRIPTION
**Pull request by whokilledtheelectricmonk**
Context of example was not clear, standalone async function vs object literal have different syntax
See #5869, #5885

### CODE REVIEW

File: docs/Network.md [hunk 1]
```diff
@@ -41,29 +41,33 @@ fetch('https://mywebsite.com/endpoint/', {
...
-    let response = await fetch('https://mywebsite.com/endpoint/');
-    return response.users;
-  } catch(error) {
-    throw error;
-  }
-}
-```
+  
+  ```js
+  var MyComponent = React.createClass({
```
Comment (satya164) [inline docs/Network.md]: Can we use the ES2015 class syntax here?

File: docs/Network.md [hunk 2]
```diff
@@ -41,29 +41,33 @@ fetch('https://mywebsite.com/endpoint/', {
...
+    ...
+    async getUsersFromApi() {
+      try {
+        let response = await fetch('https://mywebsite.com/endpoint/');
+        let responseJson = await response.json();
+        return responseJson.users;
+      } catch(error) {
+        throw error;
```
Comment (satya164) [inline docs/Network.md]: We should add a comment here `// Handle error`, and `console.error` instead of throwing.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by mkonicek** [comment]
Commit is in master but there's a bug with commit messages not containing a link back to the PR. Normally this PR would be closed automatically. Closing manually until the bug is fixed.

**Pull request closed without merging**